### PR TITLE
XRT-2795 branch

### PR DESCRIPTION
**Pull request by SteveOss**
- Updated openSUSE support from 15.4 to 15.5
- Brought doxygen file up to date
- Rolled IOT version to 1.5.2.0 as 1.5.1 now released

### CODE REVIEW

**Comment by SteveOss** [comment]
Have build development and release packages

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -385,3 +385,7 @@
 - Support added for Alpine Linux 3.19
 - Added `iot_data_length` function for determining number of elements in iot_data object
 - Added `iot_store_config_list` and `iot_file_list` functions to list all files in a given directory
+
+## Version 1.5.2
+
+- Support added for openSUSE Leap 15.5
```
Comment (BinduRao2018) [inline RELEASE_NOTES.md:391]: Should we say removed support for openSUSE 15.4?

**Review by BinduRao2018 — COMMENTED** [review]
- comment on the Release note update

File: RELEASE_NOTES.md (hunk 1 above)
Comment (SteveOss) [reply RELEASE_NOTES.md]: Updated

**Review by BinduRao2018 — APPROVED** [review]
LGTM

**Pull request merged**